etc/sample.bcoin.conf

```diff
@@ -0,0 +1,122 @@
+# # Sample bcoin config file (~/.bcoin/bcoin.conf)
+# can change this location by passing `prefix`
+# option to your node
+# If running non-mainnet node, this should be
+# in a sub-directory w/ the name of the network
+
+
+#
+# Options
+#
+
+# network: main
+
+#
+# Node
+#
+
+prefix: ~/.bcoin
+db: leveldb
+max-files: 64
+cache-size: 100
+
+#
+# Workers
+#
+
+workers: true
+# workers-size: 4
+# workers-timeout: 5000
+
+#
+# Logger
+#
+
+log-level: debug
+log-console: true
+log-file: true
+
+#
+# Chain
+#
+
+prune: false
+checkpoints: true
+coin-cache: 0
+entry-cache: 5000
+index-tx: false
+index-address: false
+
+#
+# Mempool
+#
+
+mempool-size: 100
+limit-free: true
+limit-free-relay: 15
+reject-absurd-fees: true
+replace-by-fee: false
+persistent-mempool: false
+
+#
+# Pool
+#
+
+selfish: false
+compact: true
+bip37: false
+bip151: true
+listen: true
+max-outbound: 8
+max-inbound: 30
+
+# Proxy Server (browser=websockets, node=socks)
+# proxy: foo:bar@127.0.0.1:9050
+# onion: true
+# upnp: true
+
+# Custom list of DNS seeds
+# seeds: seed.bitcoin.sipa.be
+
+# Local Host & Port (to listen on)
+host: ::
+# port: 8333
+
+# Public Host & Port (to advertise to peers)
+# public-host: 1.2.3.4
+# public-port: 8444
+
+# BIP151 AuthDB and Identity Key
+bip150: false
+identity-key: 74b4147957813b62cc8987f2b711ddb31f8cb46dcbf71502033da66053c8780a
+
+# Always try to connect to these nodes.
+# nodes: 127.0.0.1,127.0.0.2
+
+# Only try to connect to these nodes.
+# only: 127.0.0.1,127.0.0.2
+
+#
+# Miner
+#
+
+coinbase-flags: mined by bcoin
+# coinbase-address: 1111111111111111111114oLvT2,1111111111111111111114oLvT2
+preverify: false
+max-block-weight: 4000000
+reserved-block-weight: 4000
+reserved-block-sigops: 400
+
+#
+# HTTP
+#
+
+http-host: ::
+# http-port: 8332
+# ssl: true
+# ssl-cert: @/ssl/cert.crt
+# ssl-key: @/ssl/priv.key
+api-key: bikeshed
+# no-auth: false
+# cors: false
+init-script:setup-coinbase-address.js
```

Comment:
Maybe we should add a header for `bPanel` specific settings, similar to the other sections

```
#
# bPanel
#
```

Reply:
good call